src/app/app.component.ts

```diff
@@ -85,7 +86,8 @@ export class CopayApp {
     JoinWalletPage,
     PaperWalletPage,
     ShapeshiftPage,
-    WalletDetailsPage
+    WalletDetailsPage,
+    ConfirmInvoicePage
```

Comment:
Allows redirect to ConfirmInvoicePage on event "IncomingDataRedir" from incoming-data.ts